tethys_cli/db_commands.py

```diff
@@ -470,15 +482,22 @@ def process_args(args):
 
     """
     db_settings = settings.DATABASES[args.db_alias]
+
     db_dir = db_settings.get("DIR")
     if db_dir is None:
         if args.command in ["init", "start", "stop"]:
             raise RuntimeError(
                 f"The tethys db {args.command} command can only be used with local databases."
             )
     else:
-        if not Path(db_dir).is_absolute():
-            db_dir = Path(get_tethys_home_dir()) / db_dir
+        db_dir = relative_to_tethys_home(db_dir)
+
+    db_engine = db_settings.get("ENGINE")
+    if not db_engine.endswith("postgresql"):
+        if args.command == "create":
+            raise RuntimeError(
+                f"The tethys db create command can only be used with PostgreSQL databases."
+            )
```

Comment:
I'm assuming this is because the method for creating the database is postgresql specific at this moment? In the future it would be good to support creating the database in other non-sqlite dbs.

Reply:
That's right. The code that we have to create and setup the database is specific to Postgres.